views/js/recommended-modules.js

```diff
@@ -40,7 +40,7 @@ var mbo = {};
     toolbarLastElement: '.toolbar-icons a:last-of-type',
     recommendedModulesButton: '#recommended-modules-button',
     oldButton: '#page-header-desc-configuration-modules-list',
-    contentContainer: '#main-div .content-div .container:last',
+    contentContainer: '#main-div .content-div .row:last .col',
```

Comment:
Hey 🖖 , thanks for your work!

What about adding new selectors, at the same time as keeping old selector ? Selecting with a generic class is worst to me as it's linked to the layout and can change anytime

Reply:
What about making this list of selectors and this use first found?

Reply:
What do you mean ? I think this is what I said, but differently 😄

Reply:
Is current solution better?